assets/js/screens/Collection/List.jsx

```diff
@@ -20,13 +28,40 @@ const ScreensCollectionList = () => {
     }
   }, [data]);
 
+  const [deleteCollection] = useMutation(DELETE_COLLECTION, {
+    onCompleted({ deleteCollection }) {
+      addToast(`Collection ${deleteCollection.name} deleted successfully`, {
+        appearance: "success",
+        autoDismiss: true
+      });
+    },
+    refetchQueries(mutationResult) {
+      return [{ query: GET_COLLECTIONS }];
+    }
+  });
+
   if (loading) {
     return <Loading />;
   }
   if (error) {
     return <Error error={error} />;
   }
 
+  const onOpenModal = (collectionId, collectionName) => {
```

Comment:
this parameter would then look like `const onOpenModal = collectionObj => {...`

Reply:
This is a great idea, I'm a huge fan of one-liners! I'll do a re-check before i submit for PRs next time to see if I can think ES6 way!